frontend/components/product-list/sections/FilterableProductsSection/CurrentRefinements.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+   Button,
+   Collapse,
+   Flex,
+   Tag,
+   TagCloseButton,
+   TagLabel,
+} from '@chakra-ui/react';
+import { formatFacetName } from '@helpers/algolia-helpers';
+import * as React from 'react';
+import {
+   useClearRefinements,
+   useCurrentRefinements,
+} from 'react-instantsearch-hooks-web';
+
+export function CurrentRefinements() {
+   const currentRefinements = useCurrentRefinements();
+   const clearRefinements = useClearRefinements();
+
+   return (
+      <Collapse
+         in={currentRefinements.items.length > 0}
+         animateOpacity
+         data-testid="current-refinements"
+      >
+         <Flex pt="3" wrap="wrap" px={{ base: 6, sm: 0 }}>
+            {currentRefinements.items.map((item, index) => {
+               return (
+                  <React.Fragment key={item.label}>
+                     {item.refinements.map((refinement) => {
```

Comment:
Maybe this is intentional, but I noticed that the `<CurrentRefinements>` [component provided by Algolia](https://instantsearchjs.netlify.app/stories/?path=/story/refinements-currentrefinements--default) lumps refinements from the same facet into a single tag, and here we put all refinements in their own tag

![image](https://user-images.githubusercontent.com/52104630/173115438-99573be7-3371-4159-b370-a9f2f2017665.png)

![image](https://user-images.githubusercontent.com/52104630/173115484-cf227b71-0430-4977-a1c4-a233195857a2.png)

Reply:
I'm following our mockup, so it's expected to be that way. Thanks for pointing this out though, the Algolia approach it's interesting and we might want to iterate on this later to improve it 👍